packages/quick_actions/quick_actions_ios/README.md

```diff
@@ -0,0 +1,16 @@
+# quick\_actions\_ios
+
+The Android implementation of [`quick_actions`][1].
```

Comment:
iOS :)

Reply:
Thanks, that was a stupid copy/paste mistake. Fixed now.